packages/imperative/src/utilities/__tests__/CliUtils.unit.test.ts

```diff
@@ -320,6 +320,13 @@ describe("CliUtils", () => {
             CliUtils.showMsgWhenDeprecated(handlerParms);
             expect(responseErrText).toEqual(notSetYet);
         });
+
+        it("should produce alternative text when deprecatedReplacement is an empty string", () => {
```

Comment:
@t1m0thyj does it make sense to make another test that checks that this alternate text isn't present when a non-empty string is passed to showMsgWhenDeprecated? probably unnecessary

Reply:
We could add an `expect` statement to one of the tests above to verify that the output does not contain the alternate text 😋